synapse/models/infotech.py

```diff
@@ -2582,7 +2590,44 @@ def getModelDefs(self):
                     ('sandbox:file', ('file:bytes', {}), {
                         'doc': 'The initial sample given to a sandbox environment to analyze.'
                     }),
+
+                    # TODO
+                    # ('windows:task', ('it:os:windows:task', {}), {
+                    #     'doc': 'The Microsoft Windows scheduled task responsible for starting the process.'}),
+
+                    ('windows:service', ('it:os:windows:service', {}), {
+                        'doc': 'The Microsoft Windows service responsible for starting the process.'}),
+                )),
+
+                ('it:os:windows:service', {}, (
```

Comment:
Should `type`, `start`, and `errorcontrol` be enums? They have defs in the msdn article.